src/connector/src/source/cdc/enumerator/mod.rs

```diff
@@ -17,9 +17,14 @@ use async_trait::async_trait;
 use crate::source::cdc::{CdcProperties, CdcSplit};
 use crate::source::SplitEnumerator;
 
-#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub const MYSQL_CDC_PREFIX: &str = "RW_CDC_";
+
+#[derive(Debug)]
 pub struct CdcSplitEnumerator {
+    /// The source_id in the catalog
     source_id: u32,
+    /// Debezium will assign a partition identifier for each table
+    partition: String,
```

Comment:
can you explain more? maybe give an example here

Reply:
You can refer to the comment of `DebeziumOffset` in the connector node. That should be detailed enough I think.

Reply:
Yes. I think the concept of `partition` and `offset` is inherited from Kafka. Since Debezium relies on the stack of Kafka connect. Quotes from the class doc of `SourceRecord` in Kafka connect. We can rename it to be more clear.

> SourceRecords are generated by SourceTasks and passed to Kafka Connect for storage in Kafka. In addition to the standard fields in ConnectRecord which specify where data is stored in Kafka, they also include a sourcePartition and sourceOffset.
> The sourcePartition represents a single input sourcePartition that the record came from (e.g. a filename, table name, or topic-partition). The sourceOffset represents a position in that sourcePartition which can be used to resume consumption of data.

Reply:
FYI: https://kafka.apache.org/20/javadoc/org/apache/kafka/connect/source/SourceRecord.html